demo/src/app/components/datepicker/datepicker.module.ts

```diff
@@ -114,6 +116,12 @@ const DEMOS = {
     type: NgbdDatepickerConfig,
     code: require('!!raw-loader!./demos/config/datepicker-config').default,
     markup: require('!!raw-loader!./demos/config/datepicker-config.html').default
+  },
+  keyboard: {
```

Comment:
Please put the demo **before**  the one about `NgbdDatepickerConfig`... Just by convention, custom config demo is always the last one in all components . Sorry about this.